### PR TITLE
[16.0][MIG] users_ldap_populate: Migration to version 16.0

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
- Standard migration to v16
- Create folder wizard to add transient model and view

@Tecnativa

TT45119

@pedrobaeza @CarlosRoca13

### CODE REVIEW

File: users_ldap_populate/models/users_ldap.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def action_populate(self):
                         )
                         user_id = res[0]
                     else:
-                        raise UserError(
+                        raise UserError from None(
```
Comment (pedrobaeza) [inline users_ldap_populate/models/users_ldap.py]: You can capture the exception in `except AccessDenied as e:` and launch this one from there.

File: users_ldap_populate/wizards/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Copyright 2023 Tecnativa - Carolina Fernandez
```
Comment (pedrobaeza) [inline users_ldap_populate/wizards/__init__.py]: `__init__.py` files shouldn't contain copyright.

Comment (pedrobaeza) [reply users_ldap_populate/wizards/__init__.py]: You can remove the other copyright as well.

**Review by pedrobaeza — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]
LGTM :+1:

**Pull request merged**